modules/ztp-configuring-ipv6.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-configuring-ipv6_{context}"]
+= Configuring IPv6 addresses for a disconnected environment
+
+Optionally, you can configure IPV6 addresses for the managed clusters.
+
+.Procedure
```

Comment:
It's good to have prereqs that help anchor the user in the process sequence, especially since we're supposed to try to write procedures like they might be published independently, some day. This might be another improvement to consider for GA.